sedai/manifest.json

```diff
@@ -1,13 +1,13 @@
 {
   "display_name": "Sedai",
-  "maintainer": "Paul.Wujek@sedai.io",
+  "maintainer": "praveen.prakash@sedai.io",
```

Comment:
In the `CODEOWNERS` [file](https://github.com/DataDog/integrations-extras/blob/master/.github/CODEOWNERS#L102), can you update the maintainer information as well?

Reply:
@yzhan289 ,
Done. Thanks